### DISCUSSION TITLE
It is recommended to optimize the remote dialog box to maximize and minimize. The closing button is too small and easy to misoperate.

### DISCUSSION THREAD

**Discussion by youhers** [rustdesk/rustdesk#2894] [General]
At present, it is easy to make mistakes. It is too small. Especially when adding new links, the+button is not very convenient. It is convenient to put it on the right side of the page like a browser. On the right side of the page, if it is not good, it will make mistakes

**Comment by youhers** [comment]
![企业微信截图_20230121154937](https://user-images.githubusercontent.com/106003909/213850494-833346f0-f81d-43a5-9184-64fca26f42f9.png)
Sunlogin This can be referred to

**Comment by youhers** [comment]
Is there any chance to wait for your optimization

**Comment by rustdesk** [comment]
we accpt pr for this.

**Comment by youhers** [comment]
Looking forward to future optimization
